snapshot-labs/sx-monorepo
# feat: Votes tab in space user profile

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/112

This PR adds the "Latest Votes" tab in the space user profile

Some parts have been extracted into their own component for easier reuse:

- ProposalVoteChoice: to show a vote choice
- ProposalsListItemHeading: the left part of a proposal list item containing the title and metadata

### How to test

1. Go to any space user profile (access from a space dashboard page)
2. There should be a new "Latest votes" tab, with the total votes count
3. The tab should list the user's vote, with an infinite scroll (20 proposals per page)

### TODO

- [x] Add support for onchain spaces
